### ISSUE TITLE
Question: Book Versions on the Website

### ISSUE THREAD

**Issue by KrisRecoveryDharma** [RecoveryDharma/website#57]
Page:  https://recoverydharma.org/book

Hello Website Friends,

I am working on the book page for the new website. I need realize that I need more data to make an informed decision about the various links we have for the Recovery Dharma Book. We are planning to add two additional versions - chapter by chapter PDF and Insight Timer. If all of the current links are behind used, then let's keep them. but if they are not, then let's simplify. Also, if they are not consistent with the page numbers of the printed copy, then they also need to be removed. 

Here are my initial thoughts, but I need more information. What are your thoughts? 

Buy A Copy:  Keep, add a description for Kindle and Audible
Chapter by Chapter PDF: Add
Download EPub:  I could not make this work
Download Paperback PDF:  Keep, but rename and describe
Download ACWE (Kindle Compatible):  I could not make this work. If keep, then let's improve description
Download a printable copy:  Drop - does not align with pages
Listen to book on Soundcloud, Apple Podcasts, Insight Timer,- Keep, but simplify to Listen - and then use icons

Thanks!

Kris

**Comment by unity1979** [comment]
I agree with everything Kris said, plus these updates. I was able to get these things to work:
✅ Get free EPUB: Keep (Works OK on Mac)
✅ Get free Amazon Kindle Book: Keep (Works OK if Kindle App is installed)

Listen to the book and to meditations:
✅ Listen on Apple Podcasts: Keep (Works OK on Mac or iPhone)

**Comment by unity1979** [comment]
💡 ALSO: Bookreading Video! (Great for people with difficulties reading. It's Sean Kelly's reading of the audiobook sync'ed with the textbook.)  Any chance we can put up this version to help with those iwth reading disabilities? These are extremely helpful for staying focused. 

See for yourself: https://www.dropbox.com/s/r4o16ii3q5hu61v/SK%20-%20RD%20p.i-xvi%20Preface.mp4?dl=0

**Comment by unity1979** [comment]
💡For consideration: Audiobook with **Female voice (for sensitivity**): https://vocaroo.com/1P2zy8QIq837

https://www.dropbox.com/sh/yvml4cbrwrqxc9n/AAAeo4UyFI7chwQksXmCBxpda?dl=0

**Comment by tom-raley** [comment]
@unity1979 @KrisRecoveryDharma Is this issue still open? Trying to clean out our backlog of issues 😄

**Comment by unity1979** [comment]
This is open still due to Broken google play link

**Comment by unity1979** [comment]
Closing—these have been addressed on both sites (Google Play Audiobook and other broken links removed) 

<img width="442" alt="image" src="https://user-images.githubusercontent.com/70724023/153470497-97fc2d1f-3ead-418f-ac65-84715815e778.png">
